src/main/java/training/subsystems/Arm/Elbow/NeoElbow.java

```diff
@@ -0,0 +1,66 @@
+package training.subsystems.Arm.Elbow;
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class NeoElbow implements IElbow {
+
+    private final CANSparkMax motor;
+
+    private static NeoElbow instance;
+
+
+
+    public NeoElbow(){
+        this.motor = new CANSparkMax(ElbowConstants.ELBOW_ID, CANSparkLowLevel.MotorType.kBrushless);
+        motor.getEncoder().setPositionConversionFactor(ElbowConstants.ELBOW_GEAR_RATIO.getDegrees());
+        motor.getPIDController().setP(ElbowConstants.ELBOW_P);
+        motor.getPIDController().setI(ElbowConstants.ELBOW_I);
+        motor.getPIDController().setD(ElbowConstants.ELBOW_D);
+        motor.getEncoder().setPosition(ElbowConstants.ELBOW_START_POSITION.getDegrees());
+    }
+
+    public static NeoElbow getInstance() {
+        if (instance == null) {
+            instance = new NeoElbow();
+        }
+        return instance;
+    }
```

Comment:
no need instance delete it in type system

Reply:
why no need? and whats type system